web-ng/src/app/services/auth/auth.service.ts

```diff
@@ -62,7 +63,9 @@ export class AuthService {
   }
 
   isAuthenticated$(): Observable<boolean> {
-    return this.getUser$().pipe(map(user => user.isAuthenticated));
+    return this.getUser$().pipe(
```

Comment:
As mentioned in a previous PR, burying the workaround override the expected behavior of a method like we're doing here will lead to "surprises" when debugging - it would be clearer of `isAuthenticated$()` always returned whether the user is actually authenticated. Callers can always disregard this value if `environment.useEmulators` is true.